chemprop/nn/predictors.py

```diff
@@ -141,9 +139,6 @@ def n_tasks(self) -> int:
     def forward(self, Z: Tensor) -> Tensor:
         return self.output_transform(self.ffn(Z))
 
-    def train_step(self, Z: Tensor) -> Tensor:
-        return self.ffn(Z)
```

Comment:
why was this deleted?

Reply:
This is now equivalent with `forward` step because `output_transform` won't do anything during training

Reply:
I don't think this change was correct, but the PR has been merged so it can be addressed later. Broadly, the introduction of the `OutputTransform` class obviates the need for separate `forward()` and `train_step()` methods. #811 should address this by removing the method

Reply:
I agree and started to remove train_step, but then found that the more complicated FFNs like `MulticlassDirichletFFN` have more differences between forward and train_step. I also agree that #811 should address this as these are somewhat related to uncertainty.